api/importer/execute/keptn_endpoints.go

```diff
@@ -0,0 +1,7 @@
+package execute
+
+type StaticKeptnEndpointProvider struct{}
+
+func (_ StaticKeptnEndpointProvider) GetControlPlaneEndpoint() string {
+	return "http://shipyard-controller:8080"
```

Comment:
Instead of using a static value for creating the URL, the provider should probably read the `CONTROLPLANE_URI` environment variable to like it was done when creating the `ImportHandlerFunc`: 
https://github.com/keptn/keptn/blob/1b140dcc828e70543e218b628f871d67d65d79e5/api/restapi/configure_keptn.go#L111-L118

Reply:
I was thinking of having multiple implementations of the `KeptnEndpointProvider` (static one for simplicity when running side by side with keptn and other one that is configurablem maybe with smart defaults). This is the simplest implem of them all though...
I was also thinking of making sure that the ProjectChecker and the executors use the same Keptn control plane, so probably they will receive the same EndpointProvider (based on env or not :stuck_out_tongue_winking_eye:  )

Reply:
Please follow a pattern similar to
https://github.com/keptn/keptn/blob/5aa0a78dfd668c86f7a2c056882586c63d975ca6/api/utils/event.go#L17-L28

(I don't mind where you put the code though, that's your call)

Reply:
Are you referring to the `SanitizeURL` usage or accessing directly the Environment (makes testing a little harder than it needs to be)?

Reply:
both, though feel free to provide a default value if the environment variable is not set.